### PR TITLE
feat: add supbase template

### PR DESCRIPTION
**Pull request by songtianlun**

### CODE REVIEW

**Comment by songtianlun** [comment]
please to skip ci config.

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,21 +62,21 @@ jobs:
...
             else
               TAG="canary"
-              docker manifest create dokploy/dokploy:${TAG} \
-                dokploy/dokploy:${TAG}-amd64 \
-                dokploy/dokploy:${TAG}-arm64
-              docker manifest push dokploy/dokploy:${TAG}
+              docker manifest create songtianlun/dokploy:${TAG} \
+                songtianlun/dokploy:${TAG}-amd64 \
+                songtianlun/dokploy:${TAG}-arm64
+              docker manifest push songtianlun/dokploy:${TAG}
```
Comment (predragnikolic) [inline .circleci/config.yml]: Revert this change please

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -46,5 +46,5 @@ jobs:
           context: .
           file: ./Dockerfile.website
           push: true
-          tags: dokploy/website:latest
+          tags: songtianlun/dokploy-website:latest
```
Comment (predragnikolic) [inline .github/workflows/deploy.yml]: Same as above, revert this change

File: .circleci/config.yml (hunk 1 above)
Comment (songtianlun) [reply .circleci/config.yml]: ok

**Comment by Siumauricio** [comment]
Wow really thank you @songtianlun appreciated!,

Just a question we need some credentials or something in storage? I tested almost everything and it was working only the storage is the only thing that have troubles, but I don't know if the issue comes from supabase side
Screenshot 2024-08-18 at 1 01 19 PM

**Comment by songtianlun** [comment]
> Wow really thank you @songtianlun appreciated!,
> 
> Just a question we need some credentials or something in storage? I tested almost everything and it was working only the storage is the only thing that have troubles, but I don't know if the issue comes from supabase side Screenshot 2024-08-18 at 1 01 19 PM

I tried to fix some issues that might cause storage anomalies, and now the container is running abnormally, but it seems that there are still some issues with authentication. I don't have time during the week, and I will try to fix the problem some other time.

**Comment by songtianlun** [comment]
> Wow really thank you @songtianlun appreciated!,
> 
> Just a question we need some credentials or something in storage? I tested almost everything and it was working only the storage is the only thing that have troubles, but I don't know if the issue comes from supabase side Screenshot 2024-08-18 at 1 01 19 PM

i think is ok. supabase need generate some new JWT by secret

https://supabase.com/docs/guides/self-hosting/docker#generate-api-keys

**Comment by songtianlun** [comment]
图片

it is ok in my vps

**Review by Siumauricio — APPROVED** [review]
Thank you ❤️! works like a charm!

**Comment by ice45571** [comment]
I am new to Dokploy and this is my first time using it to deploy Supabase. Could you please tell me why my URL is showing a 404 error? Deploying other projects works fine. Thank you for your help.

image
http://chatgpt-supabase-a426f2-4b944f-104-168-112-236.traefik.me/

**Comment by songtianlun** [comment]
> I am new to Dokploy and this is my first time using it to deploy Supabase. Could you please tell me why my URL is showing a 404 error? Deploying other projects works fine. Thank you for your help.
> 
> image http://chatgpt-supabase-a426f2-4b944f-104-168-112-236.traefik.me/

wait a minutes or deploy again.

**Comment by ice45571** [comment]
> > I am new to Dokploy and this is my first time using it to deploy Supabase. Could you please tell me why my URL is showing a 404 error? Deploying other projects works fine. Thank you for your help.
> > image http://chatgpt-supabase-a426f2-4b944f-104-168-112-236.traefik.me/
> 
> wait a minutes or deploy again.

Thanks, clicking on the 'Domains' tab under 'General' solved my issue.

**Comment by dxqwww** [comment]
+1

I thought it would work right out of the box, but I get the message: `404 Page Not Found` when I try to access the site

**Comment by baotr-hippo** [comment]
@songtianlun Hi. Have you ever tried uploading to supabase storage? My storage is working properly and can list buckets that have been created. But uploading files is not. It looks like this API has an error `/storage/v1/upload/resumable`

**Comment by adrianbruegger** [comment]
> @songtianlun Hi. Have you ever tried uploading to supabase storage? My storage is working properly and can list buckets that have been created. But uploading files is not. It looks like this API has an error `/storage/v1/upload/resumable`

Hi @songtianlun , I’m getting the same error. It looks like an issue with the authentication. I would really appreciate it if you could take a look.

**Comment by baotr-hippo** [comment]
> > @songtianlun Hi. Have you ever tried uploading to supabase storage? My storage is working properly and can list buckets that have been created. But uploading files is not. It looks like this API has an error `/storage/v1/upload/resumable`
> 
> Hi @songtianlun , I’m getting the same error. It looks like an issue with the authentication. I would really appreciate it if you could take a look.

The problem seems related to Traefik or the SSL certificate. Upload requests via the dashboard, it will send a request to the domain of the traefik that does not have an SSL certificate. Using the Supabase SDK should solve this since it uses your Supabase URL. The JavaScript SDK works well for my custom domain.

**Comment by agusibrahim** [comment]
image
please help, always facing this issue

**Comment by songtianlun** [comment]
> image
> 
> please help, always facing this issue

please to check docker log.

**Comment by agusibrahim** [comment]
> > image
> > please help, always facing this issue
> 
> please to check docker log.

done by prune images and re-deploy

**Comment by znowfox** [comment]
so, the big issues are for multiple instance: we can't have duplicate names of containers, but if we do change realtime's container name on a second instance we then break it.

**Comment by znowfox** [comment]
so 
`    hostname: realtime-dev.supabase-realtime` added to realtime lets it work regardless of container name

and something like 
image
allows for dynamic container names, should be enough to host any amounts of supabase services
